index.markdown

```diff
@@ -370,8 +370,7 @@ redirect_from: '/moveit/'
                         <p><span>Version:</span> 2.0</p>
                         <p><span>Platforms:</span> Ubuntu 18.04</p>
                         <p><span>Initial release:</span> 14 February 2020</p>
-                        <p><span>End of life:</span> November 2020</p>
-                        <p><span>Branch:</span> <a href="https://github.com/ros-planning/moveit2" target="_blank">moveit2:master</a></p>
```

Comment:
Is there not branch for Eloquent remaining? Can we change the hyperlink to a commit ID instead, right before you merged the foxy branch?

Reply:
I pushed a branch and referenced it